### PR TITLE
example: Use SPIFFS init from BSP

### PR DESCRIPTION
**Pull request by espzav**
# ESP-BSP Pull Request checklist

> Note: For new BSPs create a PR with this [link](https://github.com/espressif/esp-bsp/compare/main...my-branch?quick_pull=1&template=pr_template_bsp.md).

- [x] Version of modified component bumped
- [x] CI passing

### CODE REVIEW

**Comment by espzav** [comment]
@tore-espressif PTAL

File: examples/display_audio_photo/main/app_disp_fs.c [hunk 1]
```diff
@@ -170,19 +170,11 @@ void app_audio_init(void)
...
-        .partition_label = NULL,
-        .max_files = 5,
-        .format_if_mount_failed = true
-    };
-
     file_buffer_size = BSP_LCD_H_RES * BSP_LCD_V_RES * sizeof(lv_color_t);
     file_buffer = heap_caps_calloc(file_buffer_size, 1, MALLOC_CAP_DEFAULT);
     assert(file_buffer);
 
     /* Initialize file system */
```
Comment (tore-espressif) [inline examples/display_audio_photo/main/app_disp_fs.c:177]: Nitpick: could you please remove this comment?

**Review by tore-espressif — APPROVED** [review]
Only one nitpick, LGTM!

**Comment by espzav** [comment]
@tore-espressif Thank you for the review!

**Pull request merged**